policies/api-compat.md

```diff
@@ -14,11 +14,15 @@ includes, but is not limited to:
 - changing a macro to a function and
 - corrections of spelling.
 
-Only API additions are allowed in minor releases.
+Only API additions are allowed in [minor] releases.
```

Comment:
```suggestion
Only API additions are allowed in [minor releases].
```

Reply:
This cascades into too many other policies and lots of voting :)

Reply:
I don't get the point. I am just suggesting to change the link _text_ in a single file. How could this cascade?

Reply:
The target is linked the same way in other files.  We should change the link text for all other instances for consistency.

Reply:
> lots of voting

There is no need to have a separate vote for every policy change. A single change that is voted on can affect multiple policy files.

Reply:
This change seems weird - the glossary item is "minor releases" not "minor"

Reply:
In the same way that you have "stable releases" below.